docs/design-docs/overview.md

```diff
@@ -72,6 +72,7 @@ Here're the details about work directory structure for SuperBench Runner.
         │       └── sb-exec.log               # collected SuperBench Executor log
         ├── sb-run.log                        # SuperBench Runner log
         ├── sb.config.yaml                    # SuperBench configuration snapshot
+        ├── pattern_config                    # generated host groups config under specified patterns for diagnosis
```

Comment:
add ".txt", and maybe rename to mpi_pattern?

Reply:
udpated.